test/cmdlineTests/model_checker_bmc_loop_iterations/err

```diff
@@ -0,0 +1,11 @@
+Warning: BMC: Assertion violation happens here.
+  --> model_checker_bmc_loop_iterations/input.sol:10:3:
+   |
+10 | 		assert(x == 3);
+   | 		^^^^^^^^^^^^^^
+Note: Counterexample:
+  x = 2
+
+Note: Callstack:
+Note: False negatives are possible when unrolling loops.
```

Comment:
Where exactly is the extra whitespace? There was an extra linebreak but looks like you fixed it already?